docs/SETUP.md

```diff
@@ -47,9 +47,13 @@ To enable XDebug to connect to the IDE on your host machine, you need to set the
 
     export LOCAL_IP=192.168.0.7
 
+A quick way and relatively reliable way to assign the correct IP address on Linux systems is to run `hostname -I` and assigning the first result to the variable:
+
+    LOCAL_IP="$(hostname -I | awk '{print $1}')" && echo ${LOCAL_IP}
```

Comment:
If this should not go into the doc then I understand. I mainly added this because my machine has 20 interfaces (mostly bridges created by the virtual machines I guess) and it is kind of a pain to find the correct IP in that mess, I figured I would not be the only one suffering from this.

Reply:
Nice tip, thanks. Could you start the Sentence with "A relatively reliable way ..."?